testgui/testgui.pro

```diff
@@ -6,7 +6,15 @@ QT += widgets
 INCLUDEPATH += ../hidapi
 HEADERS += ../hidapi/hidapi.h
 
-mac {
- SOURCES += ../mac/hid.c
- QMAKE_LFLAGS += -framework IOKit -framework CoreFoundation
+
+win32 {
+ SOURCES += ../windows/hid.c
+ LIBS += -lSetupApi
+}
+
+unix {
+  mac {
```

Comment:
No, either is fine - note I didn't update this patch in a while, but I think everything is still valid.